.github/workflows/reusable-docker-readme.yml

```
@@ -0,0 +1,63 @@
+        required: true
+        type: string
+      short_description:
+        description: 'Docker Hub short description (<=100 chars)'
+        required: true
+        type: string
+      build_logic_ref:
+        description: 'Ref for liquibase/build-logic checkout'
+        required: false
+        type: string
+        default: 'main'
+    secrets: inherit
```

actionlint (1.7.12)

[error] "secrets" section is scalar node but mapping node is expected

(syntax-check)